storage/InsertDestination.cpp

```diff
@@ -60,25 +60,28 @@ InsertDestination::InsertDestination(const CatalogRelationSchema &relation,
                                      const StorageBlockLayout *layout,
                                      StorageManager *storage_manager,
                                      const std::size_t relational_op_index,
+                                     const std::size_t query_id,
```

Comment:
I think this is really reading too much into ordering of parameters. It doesn't make any crucial difference.